docs/content/v2024.2/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -155,17 +179,55 @@ Optionally, use the **Helm Overrides** section, as follows:
         enabled: true
     ```
 
-- Select **Force Apply** if you want to override any previous overrides.
+- If you want to use ARM VMs, add the following overrides:
+
+    ```yaml
+    # Point to the aarch64 image in case multi-arch is not available.
+    Image:
+        tag: 2.23.0.0-b586-aarch64 
```

Comment:
this tag should be picked from whatever branch this document is being rendered from.